samples/word/Basics/basic-doc-assembly.yaml

```diff
@@ -0,0 +1,241 @@
+id: ciyvyru7l00014r6ydkzkwka7

```

Comment:
not sure why this happens I just opened in the editor and it does not show anything.